apps/docs/content/components/Card/index.mdx

```diff
@@ -0,0 +1,129 @@
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image showing a card with a Copilot Octicon.](https://github.com/primer/brand/assets/912236/494d440c-98d2-4634-beec-8a80caca8926)
+
+## Usage
+
+Use cards to summarize the central idea behind some content and provide a link to its corresponding detailed resource. It is important to present the information in a concise manner, avoiding lengthy and complex descriptions.
+
+Cards should always be used in groups of two or more related concepts.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/d0da491e-8895-4ba2-9840-17eef7f2dd75" />
+    <Caption>Use cards in groups to display related information.</Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/9f17706e-dfb1-4ef9-a87d-df6c8a1d3825" />
+    <Caption>
+      Don't use isolated cards. Consider using a CTABanner instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Stacked
+
+Cards should be stacked to display a list of items that are related to each other. If displaying a visual asset, do so for all cards to achieve a visually balanced layout. The visual asset should be the same type for all cards, either an icon or an image.
+
+For a better visual experience, we recommend using similar length size for the heading and description in all stacked cards.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/fced572e-c296-4a97-aaa8-3d4b357f2f6e" />
+    <Caption>
+      Use the same type of visual asset for all cards and a similar length size
+      for the content.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/c612ccf8-7c98-4213-92b9-2fdf6f0f1398" />
+    <Caption>
+      Don't use different length sizes for the title or the description.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+When using an image as the visual asset, we recommend using the same aspect ratio for all images. This will create a more balanced visual experience.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/0ec93c16-7091-4ebb-a1a0-b0626de7833f" />
+    <Caption>
+      Use the same image ratio and a similar lenght sizes for the content.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/0b8ffb2d-0753-4bdd-9165-0f30ec28a8c8" />
+    <Caption>
+      Don't use different image ratios and a different length size for the
+      content.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+Cards typically appear in groups of three or four in desktop viewports. For smaller viewports, or when more than four cards are needed, we recommend stacking the cards vertically. You can use the [Grid](/components/Grid) or the [Stack](/components/Stack) component to achieve this.
```

Comment:
I _think_ these links will break when the site gets deployed to production because of the path prefix.

You may need to use the Gatsby Link or do this instead:

```suggestion
Cards typically appear in groups of three or four in desktop viewports. For smaller viewports, or when more than four cards are needed, we recommend stacking the cards vertically. You can use the [Grid](/brand/components/Grid) or the [Stack](/brand/components/Stack) component to achieve this.
```

Reply:
It works good on production, you used them in the `Avatar` guidelines:

https://primer.style/brand/components/Avatar

mdx code: https://github.com/primer/brand/blob/main/apps/docs/content/components/Avatar/index.mdx?plain=1#L31